docs/install.md

```diff
@@ -1,13 +1,16 @@
 # Installation Guide
 
 ## Prerequisites
+
 * [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
 * [Helm 3](https://helm.sh/docs/intro/install/)
-* A Kubernetes cluster on Azure with a dedicated nodepool backed by Azure Virtual Machine Scale Set (VMSS) with at least one node (VMSS instance).
-    * The nodes should be tainted with `kubeegressgateway.azure.com/mode=true:NoSchedule` so that no other workload can land on.
-    * The nodes should be labeled with `kubeegressgateway.azure.com/mode: true` so that kube-egress-gateway DaemonSets NodeSelector can identify these.
-    * The nodes should be linux only.
-    * The nodes should be excluded from cluster autoscaler.
+* A Kubernetes cluster on Azure with a dedicated nodepool for gateway nodes with at least one node.
+  * **For public IP egress (default)**: Nodepool backed by Azure Virtual Machine Scale Set (VMSS)
+  * **For private IP egress (preview, AKS 1.34+)**: Nodepool backed by standard Azure VMs (Availability Set mode) for stable private IP assignment
```

Comment:
not availability sets. and this install guide is for unmanaged gateway node pools which isn't supported for VM based pools yet. so this should just say "for private IP egress: node pool must of type VirtualMachines and is only supported as a managed Gateway node pool" again with a link to our public docs.